src/main/java/de/tum/in/www1/artemis/repository/StudentExamRepository.java

```diff
@@ -13,5 +15,6 @@
 @Repository
 public interface StudentExamRepository extends JpaRepository<StudentExam, Long> {
 
-    List<StudentExam> findByExamId(Long examId);
+    @Query("SELECT s FROM StudentExam s WHERE s.exam.id = :#{#examId}")
+    List<StudentExam> findByExamId(@Param("examId") Long examId);
```

Comment:
You could then also remove `@Param("examId")` here

Reply:
done